recipes/hfsm2/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+from conan import ConanFile
+from conan.tools.files import copy
+from conan.tools.build import check_min_cppstd
+from conan.tools.layout import basic_layout
+from conan.tools.files import get
+
+required_conan_version = ">=2.0.16"
+
+class Hfsm2Conan(ConanFile):
+    name = "hfsm2"
+    description = "High-Performance Hierarchical Finite State Machine Framework"
+    license = "MIT"
+    topics = ("embedded", "fsm", "state-machine", "cpp" "modern-cpp", "game-development",
+              "cpp11", "embedded-systems", "template-metaprogramming", "header-only",
+              "mit-license", "fsm-library", "hierarchical-state-machine", "game-dev", "hfsm")
+    homepage = "https://hfsm.dev/"
+    url = "https://github.com/conan-io/conan-center-index"
+
+    package_type = "header-library"
+    exports_sources = "include/*"
```

Comment:
```suggestion
```
That's only required when export from source, which means, both recipe and source are the same repository. As example: https://docs.conan.io/2/tutorial/developing_packages/local_package_development_flow.html